state/execution_test.go

```diff
@@ -218,10 +239,15 @@ func TestUpdateValidators(t *testing.T) {
 
 	for _, tc := range testCases {
 		t.Run(tc.name, func(t *testing.T) {
-			err := updateValidators(tc.currentSet, tc.abciUpdates)
+			err1 := validateValidatorUpdates(tc.abciUpdates, tc.validatorParams)
```

Comment:
I'm not entirely sure. I think these two should only be called in conjunction. That was my reasoning behind testing them together, but I'm happy to change it

Reply:
probably makes sense to test them separately so we can disentangle what's causing the errors. otherwise it's easy for the tests to accidentally pass for the wrong reason.